pages/docs/modal-system.en-US.mdx

```diff
@@ -0,0 +1,71 @@
+import { Callout } from "nextra-theme-docs";
+
+# The modal system
+
+An easier way to trigger a modal without having to have a conditional state to render it in the DOM.
+
+This is handled by the function [`showModal`](https://o3-docs.openmrs.org/docs/framework-api-reference#showmodal) defined in `esm-core` that takes care of the rendering of the modal.
```

Comment:
```suggestion
The main function users of the modal system use is the [`showModal`](https://o3-docs.openmrs.org/docs/framework-api-reference#showmodal) function defined in `esm-core`, which will render the named modal.
```

Reply:
> The main function users of the modal system use is the [`showModal`](https://o3-docs.openmrs.org/docs/framework-api-reference#showmodal) function

Sorry, Ian. I don't quite understand what this means. Could you please explain what you meant?

Reply:
Maybe @denniskigen has a better suggeestion?

Reply:
I think the way you suggested instead of the "An easier way..." is perfect! 
I was referring to the suggestion below that, specifically the:
![image](https://github.com/denniskigen/o3-docs/assets/34070216/15583b28-e029-4b50-9766-a4a3b20c9e45)

Reply:
Yeah, I got that! Maybe something like:

```suggestion
To use the modal system, first your application must register the modal, as described below. Once the modal is registered, call [`showModal`](https://o3-docs.openmrs.org/docs/framework-api-reference#showmodal) with the modal name and the framework will render the modal for you.
```

Reply:
Aaah gotcha! Sorry my bad. Thanks, Ian!